src/nfa/delta.cc

```diff
@@ -59,12 +59,9 @@
     return state_post.end();
 }
 
-size_t Delta::size() const {
-    size_t size = 0;
-    for (State q = 0; q < num_of_states(); ++q) {
-        for (const SymbolPost & m: (*this)[q]) { size = size + m.size(); }
-    }
-    return size;
+Delta& Delta::operator=(const Delta& other) {
```

Comment:
It could not be default implemented when we had transitions as a member of Delta. When we no longer have that, it can be default implemented again using the member-wise copy assignment.

Reply:
Fixed.